pubpub/pubpub
# update: Citation parser input types

This PR includes three updates to the editor's citation parsing capabilities:

1. Enable PubMed ID parsing
2. Enable ISBN parsing
3. Disable @else/url parser (i.e. arbitrary URL parsing)

See Freshdesk ticket #1308 for more context.

_Test Plan_
- Visit a Pub draft
- Confirm that bibtex and DOI parsing still function as expected
- Confirm that ISBN (-10 and -13) parsing produces a citation output
- Confirm that PubMed ID parsing produces a citation output.
